assets/countries/index.js

```diff
@@ -5,7 +5,8 @@ var countriesDomestic = [
   'Guernsey',
   'Jersey',
   'Isle of Man',
-  'United Kingdom'
+  'United Kingdom',
+  'UK'
```

Comment:
Just out of interest. Couldn't we just have something like "United Kingdom (UK)" or is that worse?